<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandPlantPlace</class>
 <widget class="QWidget" name="RandPlantPlace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>随机放置</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>551</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="select_noneed"/>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>156</width>
       <height>178</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="animatedplant">
        <property name="text">
         <string>添加种植动画</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rowcol_sure">
        <property name="text">
         <string>确 认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="change_mode">
        <property name="text">
         <string>切换至选择种植模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearallplant">
        <property name="text">
         <string>一键清空全场植物</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_button">
        <property name="text">
         <string>退 出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>171</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="currentmode">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>当前为选择不种植模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rowlabel">
        <property name="font">
         <font>
          <family>等线 Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>最大列：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxrow">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="collabel">
        <property name="font">
         <font>
          <family>等线 Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>最大行：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxcol">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>请输入行列：</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>请选择不需要种植的位置（默认全部种植）：</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>380</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="reset_select">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>重 置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="finished">
        <property name="font">
         <font>
          <family>等线 Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>种植完成！</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sure_plant">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确 认 种 植</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>禁种植物</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>671</width>
       <height>401</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>669</width>
        <height>399</height>
       </rect>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>671</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="forbidarea"/>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>440</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>430</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="allselct">
        <property name="text">
         <string>全选</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetbutton">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forbidsure">
        <property name="text">
         <string>禁止种植</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
